roles/gateway_common/tasks/stop_gateway.yml

```diff
@@ -4,6 +4,8 @@
   # sudo to allow ssgconfig user to call this command, use raw instead of command to suppress ansible warning of suggesting become
   raw: sudo -u layer7 /opt/SecureSpan/Appliance/libexec/ssgconfig_launch -lifecycle ensureStopped
   register: status
+# ignore rc=79 which happens if the service has already been stopped.
+  failed_when: status.rc != 79 and status.rc != 0
```

Comment:
Can we just run a grep to check the Gateway.jar pid before the command rather than status.rc? I'm not sure 79 is the only status we will get.